docs/design/performance-benchmarking/README.md

```diff
@@ -0,0 +1,81 @@
+# Performance Benchmarking
+
+## Get All
+
+To do the performance tests to retrieve endpoints, use the
+[Suite3-Performance-Testing](https://github.com/Ed-Fi-Exchange-OSS/Suite-3-Performance-Testing).
+
+### Running Performance tests for Meadowlark
+
+- Configure Meadowlark and verify that it's running.
+- Load Sample data with the Invoke-LoadGrandBend or Invoke-LoadPartialGrandBend scripts.
```

Comment:
Do we need a script for loading Southreidge?

Reply:
It can be good for convenience